app/views/brexit_checker/show.html.erb

```diff
@@ -12,17 +12,21 @@
   end
 %>
 
+<% content_for :breadcrumbs do %>
 <div class="govuk-width-container">
   <% if @previous_page %>
     <%= render "govuk_publishing_components/components/back_link", {
       href: transition_checker_questions_path(page: @previous_page, c: criteria_keys),
     } %>
   <% else %>
     <%= render "govuk_publishing_components/components/back_link", {
-      href: "/get-ready-brexit-check",
+      href: "/transition",
```

Comment:
Besides fixing the skip link issue, this PR also seems to change the back link href from `/get-ready-brexit-check` to `/transition`.
Having tested I understand why it's been done. But could we please also update the PR description and commit message to mention this change and explain why it is happening? 🙏

Reply:
Have updated the commit message and PR - thanks for the spot :-)